meds/maker.py

```diff
@@ -1373,16 +1400,19 @@ def _load_config(self, config):
             self._joblib_threads = None
 
 
-def _psf_rec_func(output_path, psf_data, file_ids, rows, cols):
+def _psf_rec_func(output_path, psf_data, file_ids, rows, cols, colors):
     import joblib
 
-    joblib.dump(
-        [
-            psf_data[file_id].get_rec(row, col)
-            for file_id, row, col in zip(file_ids, rows, cols)
-        ],
-        output_path,
-    )
+    psfs = []
+    for file_id, row, col, color in zip(file_ids, rows, cols, colors):
+        try:
+            psf = psf_data[file_id].get_rec(row, col, color=color)
+        except TypeError:
```

Comment:
if the user sent colors but the function doesn't accept colors, we should be crashing.